.changeset/small-chairs-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"crib-deploy-environment": minor
```

Comment:
I'd argue this should be a major version jump, specially because of the new required input `aws-ecr-private-registry`

Reply:
Thanks. Fixed.